app/models/auction.rb

```diff
@@ -30,7 +30,7 @@ class Auction < ActiveRecord::Base
     rejected: 2
   }
 
-  enum published: { unpublished: 0, published: 1 }
+  enum published: { unpublished: 0, published: 1, archived: 2 }
```

Comment:
cool I think this is a nice place to put it. I kind of thought we might make this a boolean field one day, but it works well as an enum now that we are using it this way!